TraitsExpanded/Interfaces/TraitSet.cs

```diff
@@ -0,0 +1,109 @@
+﻿using System.Collections.Generic;
+using TaleWorlds.CampaignSystem;
+using TaleWorlds.Core;
+using TaleWorlds.MountAndBlade;
+
+namespace TraitsExpanded
+{
+    public abstract class TraitSet
+    {
+        public virtual List<ITrait> Traits { get; set; } = new List<ITrait>();
+        public virtual string ID { get; set; }
```

Comment:
setter for ID and other properties we want to expose should be protected

Reply:
Yup